api/nodes/serializers.py

```diff
@@ -1721,3 +1727,97 @@ def enable_or_disable_addon(self, obj, should_enable, addon_name, auth):
         if type(addon) == bool:
             addon = None
         return addon
+
+class OSFGroupsRelationshipField(RelationshipField):
+
+    def get_value(self, data):
```

Comment:
What on earth am I doing wrong that made me need to add this? Is it somehow because of the `CompoundIDField`? I tried forever to figure this out but got so stuck figured someone else could look at it and it'd be more obvious.